packages/jest-serializer-make-styles/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import { DEFINITION_LOOKUP_TABLE, RULE_CLASSNAME_INDEX, RTL_PREFIX } from '@fluentui/make-styles';
+
+export function print(val: string) {
+  const regexParts: string[] = [];
+  const regex = lookupRegex();
+  let result: RegExpExecArray | null = null;
+  while ((result = regex.exec(val))) {
+    const [name] = result;
+    const [definitions] = DEFINITION_LOOKUP_TABLE[name];
+    /**
+     * Collects all classNames present in a definition and adds it as part of a regular expression
+     * @example
+     * rules = ["r?f16th3vw", "r?frdkuqy0", "r?fat0sn40", "r?fjseox00"]
```

Comment:
```suggestion
     * rules = ["f16th3vw", "frdkuqy0", "fat0sn40", "fjseox00"]
```

`r?` is is added in regex creation few lines below as I see

Reply:
That's is just a comment example.... that's not code 😅